portal-frontend/src/app/features/notifications/notification-details/notification-details.component.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { DeepMocked, createMock } from '@golevelup/ts-jest';
+import { BehaviorSubject } from 'rxjs';
+import { CodeService } from '../../../services/code/code.service';
+import { NoticeOfIntentDocumentDto } from '../../../services/notice-of-intent-document/notice-of-intent-document.dto';
+import { NoticeOfIntentDocumentService } from '../../../services/notice-of-intent-document/notice-of-intent-document.service';
+import { NoticeOfIntentSubmissionDetailedDto } from '../../../services/notice-of-intent-submission/notice-of-intent-submission.dto';
+import { NoticeOfIntentSubmissionService } from '../../../services/notice-of-intent-submission/notice-of-intent-submission.service';
```

Comment:
Should this be a notification instead of a notice of intent?

Reply:
Dead imports removed.